.git-crypt/.gitattributes

```diff
@@ -0,0 +1,4 @@
+# Do not edit this file.  To specify the files to encrypt, create your own
+# .gitattributes file in the directory where your files are.
+* !filter !diff
+*.gpg binary
```

Comment:
Not sure what you mean by `refresh token` ...

This file was automatically created by `git-crypt`. we are using `git-crypt` to encrypt groups/k8s-infra-test-project-1896690daeb3.json. We have my public key and cblecker's public key to allow the two of us to decrypt the json file before we can run `reconcile.go`

Reply:
Ok. This makes sense if you want to use a service account and run this environments where you may not have access to the service account credentials. Alternatively you could use authenticate to google as dims or cblecker, and use your own authority to take these actions. This works if this main runs only in environments where credentials are already available (e.g. on your laptop), but it has the benefit of providing better auditing and authorization (dims shows up as the authority that made the change, dims can be removed from the administrator of a group individually) rather than sharing a credential that erases who did what, and needs to be rotated regularly to ensure that it hasn't escaped from the group of admins.

Just a suggestion, feel free to ignore or implement later.

Reply:
Ack thanks @mikedanese . yep, we have to iterate to get there (run in a k8s cluster in a pod)